src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -75,26 +77,79 @@ internal class RiskService(
     return if (superStatus == COMPLETE.name) COMPLETE.name else INCOMPLETE.name
   }
 
-  suspend fun fetchIndexOffenceDetails(crn: String): String? {
-    val activeConvictionsFromDelius = getValueAndHandleWrappedException(communityApiClient.getActiveConvictions(crn))
-    val assessmentsResponse = fetchAssessments(crn)
-    val latestAssessment =
-      assessmentsResponse.assessments?.sortedBy { LocalDateTime.parse(it.dateCompleted).toLocalDate() }?.reversed()
-        ?.firstOrNull()
-    val oasysAssessmentCompleted =
+  suspend fun fetchAssessmentInfo(crn: String): AssessmentInfo? {
+    val activeConvictionsFromDelius = try {
+      getValueAndHandleWrappedException(communityApiClient.getActiveConvictions(crn))
+    } catch (ex: Exception) {
+      log.info("No active convictions available for CRN: $crn - ${ex.message}")
```

Comment:
Not sure you need this info statement - the extractErrorCode should deal with logging as appropriate.